### PR TITLE
Xcom Armor Update (Sprites and Stats)

### PR DESCRIPTION
**Pull request by Optimism333**
## PR Summary ##
These suits probably hadn't been touched for years, and it was a damn shame, because Xcom is a cool reference in the server's big stew of sci-fi references. I partnered with Blithering to try to make these armor suits look better, be more interactive, and actually be in a place balance-wise where they could be a viable alternative to some of the armors available on the station.

### Old Mysterious Armor ###
image

### New Mysterious Armor ###
image

### Old Squaddie Armor ###
image

### New Squaddie Armor ###
image

### New Mysterious Helmet ###
image

image

### Mysterious Armor ###
Wearing the mysterious armor changes your screams to classic Xcom Ufo Defense screams. Four variants for men, three for women.

https://github.com/vgstation-coders/vgstation13/assets/81007356/87689cda-c1af-40c3-a3e1-d32e9859efb8

### Squaddie Armor ###
You can roll the sleeves up and down. Has no mechanical effect, but rolling your sleeves up makes you look cool.

image

## Additional Research/Material Cost and Stat Rebalance ##
Probably the section with the most contentious material.

- The suits now cost more research.
previously: 3 materials research
now: 3 combat research / 3 materials research

- The suits cost more materials to make: 
previously: 5000 iron / 1000 glass 
now: 5000 iron / 1250 glass / 750 plastic or plasma

- The suits have more balanced protection stats, while still being somewhat specialized (mysterious armor protects best against lasers, squaddie armor protects best against bullets):

### Mysterious Armor Stats ###
previous: (melee = 50, bullet = 10, laser = 10, energy = 0, bomb = 0, bio = 0, rad = 0), Low Slowdown
now: (melee = 50, bullet = 15, laser = 50, energy = 20, bomb = 25, bio = 0, rad = 0), Low Slowdown

### Squaddie Armor Stats ###
previous: (melee = 10, bullet = 50, laser = 10, energy = 10, bomb = 0, bio = 0, rad = 0), No Slowdown
now: (melee = 50, bullet = 50, laser = 15, energy = 5, bomb = 35, bio = 0, rad = 0), Low Slowdown

## Other Minor Changes ##
- The Judge Dredd suit, which was previously a child of the squaddie armor, inherits the new squaddie armor stats. It's very rare though, so I doubt this should be a problem.
- I restricted greys, vox, and insectoids from wearing the Dredd suit. It doesn't appear to have species-fitting sprites regardless, so I don't think that will change anything significant.
- The Judge Dredd suit is no longer a child of the squaddie armor

## Thanks for Reading ##
Please leave some feedback, if you have some. It's always appreciated.

[sprites]  [content] [tweak]

:cl:
 * tweak: Updated the sprites of the Xcom mysterious armor and squaddie armor (thanks to Blithering)
 * rscadd: The mysterious armor gives its wearer classic xcom screams
 * rscadd: Adds a mysterious helmet to pair with the armor. It will give you a ufo defense haircut
 * rscadd: You can roll the squaddie armor's sleeves up and down
 * tweak: The mysterious armor and squaddie armor cost more research and materials
 * tweak: The mysterious armor and squaddie armor have better defensive stats
 * tweak: The dredd suit has had its defensive stats buffed to mirror the new squaddie armor

### CODE REVIEW

**Comment by Optimism333** [comment]
Should probably have added a balance label, but I forgot.

To-Do: Try to add this balaclava that Blithering made for the mysterious suit

image

**Comment by jwhitak** [comment]
I'd say they'd be fine at up to 50/50/15 now that they cost plasma/uranium. Can't just snag it round start for free, got to do a little work, and it's a good (slight) upgrade to the standard security armor of 50/50/15. It covers the arms and legs compared to the standard security vest. It would encourage interdepartmental interactions! Security, going to Science to get upgrades??

**Comment by Optimism333** [comment]
> I'd say they'd be fine at up to 50/50/15 now that they cost plasma/uranium. Can't just snag it round start for free, got to do a little work, and it's a good (slight) upgrade to the standard security armor of 50/50/15. It covers the arms and legs compared to the standard security vest. It would encourage interdepartmental interactions! Security, going to Science to get upgrades??

I don't disagree with this. If other players think the stats could be a bit stronger with the additional costs, please do reply or thumb up this post.

**Comment by wotisjanitor** [comment]
I think they should be stronger as well

**Comment by Dacendeth** [comment]
Why human restricted for the dredd armor when only vox, insectoids and greys look weird in it, cant you just restrict them from wearing it instead?

**Comment by Optimism333** [comment]
> Why human restricted for the dredd armor when only vox, insectoids and greys look weird in it, cant you just restrict them from wearing it instead?

Fair. I will apply this instead

**Comment by Optimism333** [comment]
Dredd suit now restricts greyys, vox, and insectoids from wearing it, rather than just being locked to humans.

Xcom suits have had their stats buffed slightly as per Dilt's suggestion.

**Comment by pedr69** [comment]
-1 on making them need uranium/plasma
if theres mats people will just make advanced EODs

**Comment by Optimism333** [comment]
> -1 on making them need uranium/plasma if theres mats people will just make advanced EODs

That's a fair point, to be honest. Glancing over the advanced EOD's stats, I think that's more a problem with that thing being ridiculously overtuned (why does it provide 80 melee and 80 bullet protection but not have heavy slowdown?!). But I'm willing to drop the material costs for these suits a bit:

Maybe 6000 iron / 1500 glass / 750 uranium or plasma?

**Comment by Blithering** [comment]
I'd argue this is around the level of the stun revolver, something that's a slight upgrade to the standard issue, and so not really something that should require more than metal and glass to manufacture. As armour its low to nil risk of being misused, so a lockbox seems unnecessary. Perhaps it's fine to leave the costs as they currently are and just throw the stats a bone?

**Comment by Optimism333** [comment]
I dropped the material costs to: 5000 iron / 1250 glass / 750 uranium or plasma.

I think these suits are a decent enough upgrade to standard issue that they merit some special material in their construction. However, they cost half the metal and glass of the EOD suit, and use different special materials. So they should remain a viable mid-tier armor that science can print if they don't have the research for EOD suits, or have plasma/uranium to spare but not gold/silver.

The suits don't come in a lockbox, as far as I can see.

**Comment by Optimism333** [comment]
Looks like I have some icon conflicts to resolve...

**Comment by Optimism333** [comment]
Wow... I think I actually resolved them. Hooray!

**Comment by pedr69** [comment]
>mid tier

we don't really have any mid tier materials
unless a clown came in and ejected all the diamond to make doors(after the miner brought mats) you're gonna have enough mats to print everything you need
maybe plasma/plastic would count as mid tier mats since they're available onstation and are thus faster to obtain than the full mining assortment
>decent enough upgrade to the standard suite

they have minor slowdown which is why they're never used for standard armor
so far i've only seen them printed during simplemob outbreaks when theres no mats - if theres mats why not make EOD with 80% resist

the fact the squaddie armor looks cool as fuck now might change peoples minds though

**Comment by wotisjanitor** [comment]
Plastic would actually be a good material for the EOD

**Comment by Optimism333** [comment]
> > mid tier
> 
> we don't really have any mid tier materials unless a clown came in and ejected all the diamond to make doors(after the miner brought mats) you're gonna have enough mats to print everything you need maybe plasma/plastic would count as mid tier mats since they're available onstation and are thus faster to obtain than the full mining assortment
> 
> > decent enough upgrade to the standard suite
> 
> they have minor slowdown which is why they're never used for standard armor so far i've only seen them printed during simplemob outbreaks when theres no mats - if theres mats why not make EOD with 80% resist
> 
> the fact the squaddie armor looks cool as fuck now might change peoples minds though

Okay, I changed this after considering you and Blithering's feedback. The squaddie armor now requires plastic, and the mysterious armor requires plasma. So now the armors only require what we could call "mid-tier" materials that can be found on the station with a little work.

Provided you have the research to print them, of course.

Addendum: Also slightly reduced the research costs: 4 combat / 4 materials ---> 3 combat / 3 materials

**Comment by Optimism333** [comment]
Blithering has been kind enough to provide me with the finalized sprites for the helmet/balaclava to go along with the mysterious armor. I'll try to add it here before the weekend.

**Comment by Optimism333** [comment]
A helmet has been added that pairs with the mysterious armor, based on the original Xcom Ufo Defense (thanks to Blithering for the spritework). Upon equipping it, it will give you an Xcom-style haircut, one for men and one for women. There are three color variants, one will be chosen randomly when the helmet is equipped.

Since the helmet is identical to station-side security helmets in protection stats, it only requires glass and metal to print.

image

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -340,3 +340,42 @@
 	item_state = "helmet"
 	species_fit = list(INSECT_SHAPED)
 	body_parts_covered = FULL_HEAD|MASKHEADHAIR
+
+/obj/item/clothing/head/helmet/xcom
```
Comment (pedr69) [inline code/modules/clothing/head/helmet.dm:344]: new ablino meta? this has flashbang dampening unlike the EOD

Comment (Optimism333) [reply code/modules/clothing/head/helmet.dm:344]: Huh, weird. I would have assumed the EOD would also be under the "helmet" umbrella, but I guess it's not.

Even so, this new headgear is only as good or bad as a regular sec helmet statwise, and it requires some research before printing. I think it's fine.

**Pull request merged**